torch_geometric/graphgym/utils/agg_runs.py

```diff
@@ -28,7 +28,10 @@ def is_seed(s):
 
 
 def is_split(s):
-    if s in ['train', 'val', 'test']:
```

Comment:
right now, PyG is logging test metrics during `trainer.fit(...)` call. In the test, they are checking logged artifacts for the test step but it won't be available because the `test_step` doesn't run during trainer.fit call.

@rohitgr7 @borda

Reply:
author suggested to remove test during training